qml/FileListPage.qml

```diff
@@ -23,26 +23,59 @@ import Sailfish.Office.Files 1.0
 
 Page {
     id: page
-    property alias model: listView.model;
+    property alias model: filteredModel.sourceModel;
     property string title: "";
     property QtObject provider;
 
     allowedOrientations: Orientation.All;
 
+    FilterModel {
+        id: filteredModel
+        filterRegExp: RegExp("")
+    }
+
     SilicaListView {
         id: listView;
         anchors.fill: parent
 
         currentIndex: -1;
+        model: filteredModel
 
         children: ScrollDecorator { }
-        header: PageHeader {
+        header: SearchPageHeader {
+            id: header
+            width: parent.width
+
             //: Application title
             //% "Documents"
             title: qsTrId("sailfish-office-he-apptitle");
-
             // TODO: uncomment once there are more document sources
             // title: page.title;
+
+            //: Document search field placeholder text
+            //% "Search documents"
+            placeholderText: qsTrId("sailfish-office-tf-search-documents")
```

Comment:
I don't know. I've followed how this is done in the media player. I guess the SearchHeaderPage component there has a vocation to be public one time, or at least reusable from other QML code, so the labels and place holder texts stays on the user side instead of in the component itself. I've found a similar SearchHeaderPage component in the Sailfish/Silica/Pickers directory for instance. So I've currently duplicated the SearchHeaderPage code, trying to keep it generic in case, at one point, one can switch it with one provided in Silica…

Besides, I can move this text to the component itself if you think it's clearer for now.